src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/books';
+import categoryReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: bookReducer,
+  categories: categoryReducer,
+});
+
+const configStore = () => configureStore({ reducer: rootReducer });
```

Comment:
- [ ] [Optional] In `redux-toolkit` you don't need `combineReducer` you can directly configure your store like that:
```javascript
const store= configureStore({
  books: bookReducer,
  categories: categoryReducer,
});
```